### PR TITLE
Updated My Education Page

### PR DESCRIPTION
**Pull request by pallavigawai**
* My Education page layout- column
* Home page layout

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -73,37 +77,84 @@
...
+        <div class="col-sm-5 col-md-8 col-lg-8">
           <div class="tab-content" id="nav-tabContent">
             <div
               class="tab-pane fade show active"
               id="nav-home"
               role="tabpanel"
               aria-labelledby="nav-home-tab"
             >
-              home
+              <!-- <ul class="details">
```
Comment (chetangawai) [inline index.html]: Remove commented code

File: style.css [hunk 2]
```diff
@@ -7,47 +7,48 @@
...
   width: max-content;
-  border: 5px solid rgb(107, 7, 24);
-  background-color: rgb(2, 12, 21);
+  margin-top: 30px;
+  padding: 25px;
   width: 100%;
   height: 100%;
 }
-/* .col-5 {
-  border: 5px solid rgb(107, 7, 24);
```
Comment (chetangawai) [inline style.css:27]: Remove commented code

File: style.css [hunk 3]
```diff
@@ -7,47 +7,48 @@
...
-.col-10 {
-  border: 5px solid green;
+.container {
+  /* margin-top: 50px; */
+  background-color: black;
 }
 
-.profile {
+.nav-link {
+  /* margin-bottom: 20px; */
```
Comment (chetangawai) [inline style.css]: Remove this line

File: style.css [hunk 4]
```diff
@@ -7,47 +7,48 @@
...
-.col-4,
-.col-5,
-.col-6,
-.col-7,
-.col-8,
-.col-9,
-.col-10 {
-  border: 5px solid green;
+.container {
+  /* margin-top: 50px; */
```
Comment (chetangawai) [inline style.css]: REmove this

**Comment by pallavigawai** [comment]
1. Added Experience details into Education Page.
2.  Added Social media Icons to column first at bottom along with bootstrap link.

**Pull request merged**